src/datanode/src/datanode.rs

```diff
@@ -347,11 +346,22 @@ impl DatanodeBuilder {
         while let Some(table_value) = table_values.next().await {
             let table_value = table_value.context(GetMetadataSnafu)?;
             for region_number in table_value.regions {
+                // Augments region options with wal options.
+                // TODO(niebayes): fetch wal options map from region info.
+                // let wal_options_map = &table_value.region_info.wal_options_map;
```

Comment:
Do we need to wait for another PR to fix this `TODO`?

Reply:
Since the meta srv skeleton PR was merged, this TODO is removed.